shared/desktop/app/installer.desktop.js

```diff
@@ -1,59 +1,92 @@
 // @flow
 import * as SafeElectron from '../../util/safe-electron.desktop'
+import fs from 'fs'
+import path from 'path'
 import exec from './exec.desktop'
 import {keybaseBinPath} from './paths.desktop'
 import {quit} from './ctl.desktop'
-import {isLinux, isWindows} from '../../constants/platform'
+import {isDarwin} from '../../constants/platform'
 import logger from '../../logger'
-import UserData from './user-data.desktop'
 
-import type {InstallResult} from '../../constants/types/rpc-gen'
+const file = path.join(SafeElectron.getApp().getPath('userData'), 'installer.json')
 
-// Copied from old constants/favorite.js
-//
-// See Installer.m: KBExitFuseKextError
-const ExitCodeFuseKextError = 4
-// See Installer.m: KBExitFuseKextPermissionError
-const ExitCodeFuseKextPermissionError = 5
-// See Installer.m: KBExitAuthCanceledError
-const ExitCodeAuthCanceledError = 6
+const loadHasPrompted = () => {
+  try {
+    const data = fs.readFileSync(file, 'utf8')
+    return JSON.parse(data).promptedForCLI
+  } catch (err) {
+    if (err && err.code === 'ENOENT') {
+      console.log('[Installer] loadHasPrompted: No installer.json file')
+    } else {
+      console.warn('[Installer] loadHasPrompted: Error loading state:', err)
+    }
+    return false
+  }
+}
 
-type State = {
```

Comment:
we just need to check if its false and set it to try and save it so we don't really keep this around at a top level